modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/daemon/DelayForPeers.scala

```diff
@@ -0,0 +1,60 @@
+package org.tessellation.sdk.infrastructure.snapshot.daemon
+
+import cats.effect.Async
+import cats.syntax.applicative._
+import cats.syntax.eq._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import scala.concurrent.duration.DurationInt
+
+import org.tessellation.cli.AppEnvironment
+import org.tessellation.cli.AppEnvironment.Dev
+import org.tessellation.schema.node.NodeState.Ready
+import org.tessellation.schema.peer.Peer
+import org.tessellation.sdk.domain.cluster.storage.ClusterStorage
+import org.tessellation.sdk.domain.snapshot.DownloadDelay
+
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.numeric.PosInt
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+import retry.RetryPolicies.{constantDelay, limitRetriesByCumulativeDelay}
+import retry.retryingOnFailures
+
+object DelayForPeers {
```

Comment:
maybe PeerDiscoveryDelay?

Reply:
`PeerDiscoveryDelay` is the trait, so this has been renamed `SelectablePeerDiscoveryDelay`.

Reply:
What does it mean that it's "Selectable"? How can I select peers?

Reply:
A "selectable" peer is a responsive peer in the `Ready` state.

Reply:
ok, makes sense :+1: